ocs_ci/deployment/deployment.py

```diff
@@ -971,6 +971,21 @@ def deploy_ocs(self):
             # Change registry backend to OCS CEPHFS RWX PVC
             registry.change_registry_backend_to_ocs()
 
+        # Enable console plugin
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if (
+            ocs_version >= version.VERSION_4_9
+            and config.ENV_DATA["enable_console_plugin"]
+        ):
+            logger.info("Enabling console plugin")
+            ocp_obj = ocp.OCP()
+            patch = f'\'[{{"op": "add", "path": "/spec", "value": {"plugins": ["odf-console"]}}}]\''
```

Comment:
Maybe I am blind but do you expand some variable here with f-string? As I don't see any, I think that `f` should be deleted and if you escaping `{` by `{{` the whole command should be modified as well I think.

Reply:
we are not expanding variable. updated in latest commit